BPA/Ch.04/GLM0.stan

```diff
@@ -0,0 +1,39 @@
+data {
+  int<lower=0> nobs;            // Number of observed data
+  int<lower=0> nmis;            // Number of missing data
+  int<lower=0> nyear;           // Number of years
+  int<lower=0> nsite;           // Number of sites
+  int<lower=0> obs[nobs];       // Observed counts
+  int<lower=0> obsyear[nobs];   // Years in observed data
+  int<lower=0> obssite[nobs];   // Sites in observed data
+  int<lower=0> misyear[nmis];   // Years in missing data
+  int<lower=0> missite[nmis];   // Sites in missing data
+}
+
+parameters {
+  real alpha;
+}
+
+transformed parameters {
+  real log_lambda[nyear, nsite];
+
+  for (i in 1:nyear)
+    for (j in 1:nsite)
+      log_lambda[i, j] <- alpha;
```

Comment:
I don't see why `log_lambda` is being defined as `alpha` here.  Why not just use `alpha` in the model directly?